cmake/modules/RootConfiguration.cmake

```diff
@@ -663,8 +669,13 @@ set(ROOT_LIBRARY_DIR_SETUP "
 # ROOT configured for the install with relative paths, so use these
 get_filename_component(ROOT_LIBRARY_DIR \"\${_thisdir}/${ROOT_CMAKE_TO_LIB_DIR}\" ABSOLUTE)
 ")
-set(ROOT_BINARY_DIR_SETUP "
+set(ROOT_BINDIR_SETUP "
 # ROOT configured for the install with relative paths, so use these
+get_filename_component(ROOT_BINDIR \"\${_thisdir}/${ROOT_CMAKE_TO_BIN_DIR}\" ABSOLUTE)
+")
+# Deprecated value ROOT_BINARY_DIR
+set(ROOT_BINARY_DIR_SETUP "
+# Deprecated value, please don't use it and use ROOT_BINDIR instead.
 get_filename_component(ROOT_BINARY_DIR \"\${_thisdir}/${ROOT_CMAKE_TO_BIN_DIR}\" ABSOLUTE)
```

Comment:
Can we set this to `ROOT_BINDIR` to clarify that they are the same?

Reply:
Done, thank you!